governance/default.go

```diff
@@ -1027,6 +1028,14 @@ func (gov *Governance) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+func (gov *Governance) InitGovCache() {
+	gov.initializeCache(gov.ChainConfig)
+}
+
+func (gov *Governance) InitLastGovStateBlkNum() {
+	atomic.StoreUint64(&gov.lastGovernanceStateBlock, 0)
+}
+
```

Comment:
Can we merge both functions? In the current PR, they always go together.

Reply:
I deliberately and thoughtfully kept them separate, considering the possibility of future use.  Do you believe merging would be a better option

Reply:
If you expect future use, then I'm not against.